App crashed while trying restoring ReceiveFragment

Log
```
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property coin has not been initialized
        at io.horizontalsystems.bankwallet.modules.receive.ReceiveFragment.onCreate(ReceiveFragment.kt:38)
        at android.support.v4.app.Fragment.performCreate(Fragment.java:2414)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1418)
        at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at android.support.v4.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3269)
        at android.support.v4.app.FragmentManagerImpl.dispatchCreate(FragmentManager.java:3223)
        at android.support.v4.app.FragmentController.dispatchCreate(FragmentController.java:190)
        at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:369)
        at android.support.v7.app.AppCompatActivity.onCreate(AppCompatActivity.java:85)
        at io.horizontalsystems.bankwallet.BaseActivity.onCreate(BaseActivity.kt:26)
        at io.horizontalsystems.bankwallet.modules.main.MainActivity.onCreate(MainActivity.kt:18)
        at android.app.Activity.performCreate(Activity.java:7003)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1230)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2899)
```

--- a/app/src/main/java/io/horizontalsystems/bankwallet/modules/receive/ReceiveFragment.kt
+++ b/app/src/main/java/io/horizontalsystems/bankwallet/modules/receive/ReceiveFragment.kt
@@ -27,7 +27,7 @@ class ReceiveFragment : DialogFragment() {
 
     private lateinit var viewModel: ReceiveViewModel
 
-    private lateinit var coin: Coin
+    private var coin: Coin? = null
 
     private var itemIndex = 0
 
@@ -35,7 +35,7 @@ class ReceiveFragment : DialogFragment() {
         super.onCreate(savedInstanceState)
 
         viewModel = ViewModelProviders.of(this).get(ReceiveViewModel::class.java)
-        viewModel.init(coin)
+        coin?.let { viewModel.init(it) } ?:  dismiss()
     }
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {

